src/main/java/com/game/roullet/service/RoomService.java

```diff
@@ -0,0 +1,54 @@
+package com.game.roullet.service;
+
+import com.game.roullet.entity.Registration;
+import com.game.roullet.entity.Room;
+import com.game.roullet.repository.PlayerRepository;
+import com.game.roullet.repository.RegistrationRepository;
+import com.game.roullet.repository.RoomRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+
+@Service
+public class RoomService {
+
+    private final RoomRepository roomRepository;
+    private final PlayerRepository playerRepository;
+    private final RegistrationRepository registrationRepository;
+
+    @Autowired
+    public RoomService(RoomRepository roomRepository, PlayerRepository playerRepository, RegistrationRepository registrationRepository) {
+        this.roomRepository = roomRepository;
+        this.playerRepository = playerRepository;
+        this.registrationRepository = registrationRepository;
+    }
+
+    public int createRoom(int playerId) {
+        playerRepository.findById(playerId).get();
```

Comment:
Please change this line to get more meaningful with this:
        if (!playerRepository.findById(playerId).isPresent()) {
            throw new RuntimeException("This player with id : " + playerId + " does not exist!");
        }